hangman.py

```diff
@@ -0,0 +1,173 @@
+# Problem Set 2, hangman.py
+# Name: Sinelnyk Glib
+# Collaborators: -
+# Time spent: more time
+
+
+import random
+import string
+
+UNKNOWN_WORD = "words.txt"
+VOWELS = {"a", "e", "o", "i", "u"}
+HINTS = "*"
```

Comment:
I meant not to change Filename - better you return it, I meant to change "lower_space" - to "unknown_letter"